assets/app/scripts/controllers/overview.js

```diff
@@ -86,10 +109,12 @@ angular.module('openshiftConsole')
 
       $scope.emptyMessage = "No services to show";
       updateFilterWarning();
+      updateTopologyLater();
       Logger.log("services (list)", $scope.services);
     }));
 
     watches.push(DataService.watch("routes", $scope, function(routes) {
+      $scope.routes = {};
```

Comment:
`$scope.routes = routes.by("metadata.name")` and then iterate over that to fill in `$scope.displayRouteByService`.